test/data/instructions_with_path.yml

```diff
@@ -0,0 +1,14 @@
+---
+:class: ''
+:url: http://majles.marsad.tn/2014/fr/assemblee
+:path: members/id=212
+:to:
```

Comment:
The tests don't catch it because they don't call `.to_h` on those particular instructions. There is already a test for a missing `to_h`. I've corrected the YAML field to avoid confusion.